android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Wed Jul 22 14:26:15 IDT 2020
```

Comment:
why do you belive these gradle files should be included in the repo?

Reply:
These are probably auto-generated when running the example project. Do you think they are not necessary?

Reply:
I think they are not necessary